docker-compose/database/db-changes/users/0003_increase_task_msg.sql

```diff
@@ -0,0 +1,2 @@
+alter table events modify message text;
```

Comment:
I still don't really understand what is the purpose of this timestamp ?

Reply:
I could remove it. It was to link a job with an import after having launching it.

Reply:
I will remove it